src/managers/WebhookManager.js

```diff
@@ -147,6 +150,10 @@ class WebhookManager {
 
   _onNewWebhookEvent = (webhook: Webhook): (event: Event) => void =>
     (event: Event) => {
+      if (this._useCluster && event.fromMaster) {
```

Comment:
I think if you make master the only thing that can broadcast messages, you won't need this.